scrapeCourse.py

```diff
@@ -0,0 +1,140 @@
+import os
+import json
+import getpass
+import django
+import json
+import re
+from bs4 import BeautifulSoup
+import requests
+
+def getCourseData(code, faculty):
+  base_url_no = "https://www.ntnu.no/studier/emner/" + code
+  data_no = requests.get(url=base_url_no)
+  soup_no = BeautifulSoup(data_no.text, 'html5lib')
+
+  if (soup_no.find_all('div', {'id': 'course-details'})[0].h1.get_text() != "\n                Ingen info for gitt studieår\n            "):
+    base_url_eng = "https://www.ntnu.edu/studies/courses/" + code
+    data_eng = requests.get(url=base_url_eng)
+    soup_eng = BeautifulSoup(data_eng.text, 'html5lib')
+    facts_about_course = soup_no.findAll('div', {'class': 'card-body'})[1].p.get_text().split(":")
+    credit = -1
+    try :
+      credit = float(facts_about_course[2].split("\n")[2][20:24])
+    except ValueError:
+      print("Not valid number, number is: ", facts_about_course[2].split("\n")[2])
+    norwegianNameRaw = soup_no.title.get_text().split("-")[1]
+    norwegianName = norwegianNameRaw[1:len(norwegianNameRaw)-1]
+    englishNameRaw = soup_eng.title.get_text().split("-")[1]
+    englishName = englishNameRaw[1:len(englishNameRaw)-1]
+    courseLevelText = facts_about_course[3].split("\n")[0][1:len(facts_about_course[3].split("\n")[0])]
+    study_level = courseLevel(courseLevelText, code)
+    last_year_taught = 0
+    taught_from = 2008
```

Comment:
Why is this set to 2008?

Reply:
Every course on "emner" starts from 2008, but I know some courses started before that but we cannot get the data. taught_from is not used anywhere so maybe we could remove it